models/yang/sonic/sonic-tam-drop-monitor.yang

```diff
@@ -92,6 +120,9 @@ module sonic-tam-drop-monitor {
 						error-app-tag "Invalid Drop Monitor Sampling Rate.";
```

Comment:
Not part of your change. But, I think the error-message would be fit here instead of error-app-tag because it describes a error condition.